lib/http-bridge/test/integration/Test/Integration/Framework/Request.hs

```diff
@@ -143,6 +148,12 @@ request (Context _ (base, manager) _ _) (verb, path) reqHeaders body = do
         -- TODO: decode API error responses into ClientError
         s -> (s, Left $ decodeFailure res)
 
+    handleException = \case
+        e@InvalidUrlException{} ->
+            throwM e
+        HttpExceptionRequest _ e ->
+            return (status500, Left (HttpException e))
+
```

Comment:
Makes it possible to pattern match on HttpException failures, these occur when for instance, there's a connection failure because the target server isn't up there yet. It allows us to retry calls when polling.